manifests/t/Team/Viewer/11/Team.Viewer.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Team.Viewer
+PackageVersion: 11
+Installers:
+- InstallerUrl: https://dl.teamviewer.com/download/version_11x/TeamViewerQS-idcz3nxbq4.exe

```

Comment:
I would suggest using the vanity URL for this, to ensure it remains up to date, as remote access tools are particularly vulnerable to security risks. In fact, launching TeamViewerQS 11 doesn't even generat an ID or Password to allow control.
```suggestion
- InstallerUrl: https://dl.teamviewer.com/download/TeamViewerQS.exe
```